### PR TITLE
Fix/si 254/fix js url builder

### PR DESCRIPTION
**Pull request by alexeyvy**
Fixes this: https://oat-sa.atlassian.net/browse/NCSUP-182

### CODE REVIEW

File: views/js/util/url.js [hunk 1]
```diff
@@ -176,7 +176,17 @@ define([
                         if(!_.isEmpty(acc) || hasQueryString){
                             acc += '&';
                         }
-                        acc += encodeURIComponent(key) + '=' + encodeURIComponent(value);
+                        if (typeof value === "object") {
+                            for (var objectProperty in value) {
```
Comment (hutnikau) [inline views/js/util/url.js]: You can use `_.forOwn(obj, function(value, key) { } );` to iterate through object properties.

File: views/js/util/url.js [hunk 2]
```diff
@@ -176,7 +176,14 @@ define([
                         if(!_.isEmpty(acc) || hasQueryString){
                             acc += '&';
                         }
-                        acc += encodeURIComponent(key) + '=' + encodeURIComponent(value);
+                        if (typeof value === "object") {
+                            _.forOwn(value, function(parameterName, parameterValue) {
+                                parameterName = encodeURIComponent(key) + "[" + encodeURIComponent(parameterName) + "]";
```
Comment (krampstudio) [inline views/js/util/url.js]: do not reassign the `parameterName` variable : 
```
_.forOwn(value, function(parameterName, parameterValue) {
    acc += encodeURIComponent(key) + "[" + encodeURIComponent(parameterName) + "]=" + encodeURIComponent(parameterValue) + "&";
});
```

Comment (alexeyvy) [reply views/js/util/url.js]: done

File: views/js/util/url.js (hunk 1 above)
Comment (alexeyvy) [reply views/js/util/url.js]: done

**Pull request merged**